definition_validator/validator.cc

```diff
@@ -901,6 +905,28 @@ class ValidateWalk {
         }
     }
 
+    core::AutocorrectSuggestion suggestMethodDefinition(const core::GlobalState &gs,
+                                                        const core::Loc classOrModuleDefLoc,
+                                                        const core::MethodRef abstractMethodRef) {
+        auto [_, indentLength] = classOrModuleDefLoc.findStartOfLine(gs);
+        string indent(indentLength, ' ');
+        auto showOptions = core::ShowOptions().withUseValidSyntax().withConcretizeIfAbstract();
+
+        auto methodDefinition = core::source_generator::prettyTypeForMethod(
+            gs, abstractMethodRef, nullptr, abstractMethodRef.data(gs)->resultType, nullptr, showOptions);
+
+        vector<string> reindentedMethodDefinitionLines;
+        absl::c_transform(absl::StrSplit(methodDefinition, "\n"), std::back_inserter(reindentedMethodDefinitionLines),
+                          [indent](auto &line) -> string { return absl::StrCat(fmt::format("{}  ", indent), line); });
```

Comment:
out of curiosity, is the StrCat redundant here? could it not also be part of the fmt::format call?

Reply:
Fixed in [33c914e](https://github.com/sorbet/sorbet/pull/7048/commits/33c914e1339489f38de8736bee249a19d221d2b7).